config/runtimes/kserve-huggingfaceserver-multinode.yaml

```diff
@@ -0,0 +1,180 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterServingRuntime
+metadata:
+  name: kserve-huggingfaceserver-multinode
+spec:
+  annotations:
+    prometheus.kserve.io/port: "8080"
+    prometheus.kserve.io/path: "/metrics"
+  supportedModelFormats:
+    - name: huggingface
+      version: "1"
+      autoSelect: true
+      priority: 2
+  protocolVersions:
+    - v2
+    - v1
+  containers:
+    - name: kserve-container
+      image: kserve/huggingfaceserver:latest
+      command: ["bash", "-c"]
+      args:
+        - |
+          ray start --head --disable-usage-stats --include-dashboard false 
+          # wait for other node to join
+          until [[ $(ray status | grep -c node_) -eq ${PIPELINE_PARALLEL_SIZE} ]]; do
+            echo "Waiting..."
+            sleep 1
+          done
+          ray status
+
+          export MODEL=${MODEL_ID}
+          if [[ ! -z ${MODEL_DIR} ]]
+          then
+            MODEL=${MODEL_DIR}
+          fi
+
+          python3 -m huggingfaceserver --model_name=${MODEL_NAME}  --model_dir=${MODEL} --tensor-parallel-size=${TENSOR_PARALLEL_SIZE} --pipeline-parallel-size=${PIPELINE_PARALLEL_SIZE}
+
+      env:
+        - name: RAY_PORT
+          value: "6379"
+        - name: RAY_ADDRESS
+          value: 127.0.0.1:6379
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: VLLM_CONFIG_ROOT
+          value: /tmp      
+        - name: HF_HUB_CACHE
+          value: /tmp
+      resources:
+        requests:
+          cpu: "2"
+          memory: 6Gi
+        limits:
+          cpu: "4"
+          memory: 12Gi
```

Comment:
Yes, that's my understanding as well, but previous feedback suggested reducing it (originally 48Gi and 16 CPU). To help users adjust this part, we could include guidance in the documentation, particularly in the "Considerations" section. If a user wants to try using this multi-node feature, they should be mindful of resource allocation, so it would be helpful to provide documentation on this.